website/docs/modules/ROOT/partials/Publishing_Footer.adoc

```diff
@@ -42,6 +95,84 @@ export MILL_PGP_SECRET_BASE64=...
 export MILL_PGP_PASSPHRASE=...
 ----
 
+=== Upload Public Key for Artifacts Verification
+
+Sonatype Central Portal needs your public key to be uploaded so it can use for verification of artifacts from their end.
+
+
+[source.sh]
+----
+# Send your public key to ubuntu server so Sonatype Maven Central can use for verification of the artifacts
+gpg --keyserver  keyserver.ubuntu.com --send-keys longIDHere
+
+# Check the server for information about the public key. information will be displayed if found
+gpg --keyserver  keyserver.ubuntu.com --recv-keys longIDHere
+----
+
+=== Publishing
+
+You can publish all eligible modules in your Mill project using 
+`mill.scalalib.SonatypeCentralPublishModule/`:
+
+[source,console]
+----
+> mill mill.scalalib.SonatypeCentralPublishModule/
+----
+
+You can also specify individual modules you want to publish in two ways:
+
+[source,console]
+----
+> mill foo.publishSonatypeCentral
+----
+
+[source.console]
+----
+> mill mill.scalalib.SonatypeCentralPublishModule --publishArtifacts foo.publishArtifacts
+----
+
+=== Publishing Using Github Actions
+
+
+To publish on Github Actions, you can use something like this:
+
+```yaml
+# .github/workflows/publish-artifacts.yml
+name: Publish Artifacts
+on:
+  push:
+    tags:
+      - '**'
+  workflow_dispatch:
+jobs:
+  publish-artifacts:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - run: ./mill mill.scalalib.SonatypeCentralPublishModule/
+        env:
+          MILL_PGP_PASSPHRASE: ${{ secrets.MILL_PGP_PASSPHRASE }}
+          MILL_PGP_SECRET_BASE64: ${{ secrets.MILL_PGP_SECRET_BASE64 }}
+          MILL_SONATYPE_PASSWORD: ${{ secrets.MILL_SONATYPE_PASSWORD }}
+          MILL_SONATYPE_USERNAME: ${{ secrets.MILL_SONATYPE_USERNAME }}
+```
+
+Where `MILL_PGP_PASSPHRASE`, `MILL_PGP_SECRET_BASE64`, `MILL_SONATYPE_PASSWORD`, and
+`MILL_SONATYPE_USERNAME` configured for the repository's or organization's Github Actions
+workflows. See
+https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions[Using Secrets in Github Actions]
+for more details.
+
+
+== Instructions for Publishing to Maven Central via Legacy OSSHR (Deprecated)
```

Comment:
This should start with a paragraph explaining why this is deprecated and providing links referencing external websites (e.g. the osshr sunset page)